internal/process/provisioning/create_runtime_resource_step.go

```diff
@@ -81,9 +83,17 @@ func (s *CreateRuntimeResourceStep) Run(operation internal.Operation, log logrus
 	runtimeResourceName := steps.KymaRuntimeResourceName(operation)
 	log.Infof("KymaResourceName: %s, KymaResourceNamespace: %s, RuntimeResourceName: %s", kymaResourceName, kymaResourceNamespace, runtimeResourceName)
 
+	// get plan specific values (like zones, default machine type etc.
+	values, err := provider.GenerateValues(&operation, s.config.MultiZoneCluster, s.config.DefaultTrialProvider, s.useSmallerMachineTypes, s.trialPlatformRegionMapping, s.config.DefaultGardenerShootPurpose)
```

Comment:
Can we name `GenerateValues` and `values` variable to name based on naming that variabled `runtimeConfiguration`?

Reply:
I agree that `GenerateValues` is quite vague. But `runtimeConfiguration` seems (IMHO) not appropriate. We need better name.
The comment is quite precise - we calculate/establish plan specific values. 
Let's discuss it directly.

Reply:
Addressed in the next commit